src/picongpu/include/fields/numericalCellTypes/EMFCenteredCell.hpp

```diff
@@ -23,122 +23,139 @@
 #pragma once
 
 #include "simulation_defines.hpp"
+#include "fields/Fields.def"
 #include "math/Vector.hpp"
 
 namespace picongpu
 {
-using namespace PMacc;
 namespace emfCenteredCell
 {
+namespace traits
+{
+    /** \tparam floatD_X position of the component in the cell
+     *  \tparam DIM3     Fields (E/B/J) have 3 components, even in 1 or 2D ! */
```

Comment:
maybe the closing tag on a new line to support better readability without code highlighting